### PR TITLE
Icon addition

### PR DESCRIPTION
**Pull request by hefaestos**
# Description

Added and linked icons for: AlcoDroid, Carrot, Daylio, Star Walk 2, Two Dots

### Icons added
* AlcoDroid (`org.M.alcodroid`)
* CARROT (`com.grailr.carrotweather`)
* Star Walk 2 (`com.vitotechnology.StarWalk2Free`)
* Two Dots (`'com.weplaydots.twodotsanroid`)

### Icons updated
* Daylio (`net.daylio`)

### Icons linked
* AlcoDroid (linked `org.M.alcodroid` to `alcodroid.svg`)
* CARROT (linked `com.grailr.carrotweather` to `carrot.svg`)
* Star Walk 2 (linked `com.vitotechnology.StarWalk2Free` to `star_walk_2.svg`)
* Two Dots (linked `'com.weplaydots.twodotsanroid` to `two_dots.svg`)

## Contributor's checklist
- [x] I have followed the [Lawnicons Guidelines](./github/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/daylio.svg
Comment (KTibow) [inline svgs/daylio.svg:1]: this is an inkscape svg and doesn't have a proper width/height

**Review by KTibow — COMMENTED** [review]
all of these:
- are inkscape svgs instead of plain svgs
- have the wrong width/height
- don't have the proper padding as required

**Comment by hefaestos** [comment]
Ah, thanks - I'd misunderstood the icon size vs canvas size. If I increase the canvas size to 192x192 (currently 160x160) without scaling the art, does that also address the "proper padding" issue? (I would think it would, but want to make sure I've not misunderstood a separate thing.)

**Comment by KTibow** [comment]
Nope, you would also need to increase the viewport and properly center/size the icons

File: svgs/daylio.svg
Comment (KTibow) [reply svgs/daylio.svg:1]: also isn't this supposed to be outlined, not filled

**Comment by hefaestos** [comment]
Apologies - not clear what "viewport" is distinct from "canvas size"... 
two_dots

**Comment by KTibow** [comment]
assuming it fits exactly into the content area, it looks good to me

**Comment by hefaestos** [comment]
appreciate it! will close and fix.

**Pull request closed without merging**